.github/workflows/execute_notebooks.yml

```
@@ -0,0 +1,91 @@
+    branches: [ main ]
+    paths:
+      - 'docs/Data Formatting/*ipynb'
+      - 'docs/*ipynb'
+  workflow_dispatch:  # Allow manual trigger
+
+jobs:
+  execute-notebooks:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)